### ISSUE TITLE
V3 Conformance Updates: CLI Plugin "First Steps" Infrastructure

### ISSUE THREAD

**Issue by zFernand0** [zowe/community#1708]
As a Zowe CLI plugin user, I may need help with getting started after installing a new plugin. 

For this purpose, the CLI squad is introducing a new Imperative Configuration property that allows plugin developers to provide such information.

Q: Should this be a _best practice_ for V3 conformance criteria ?

_Originally posted by @zFernand0 in https://github.com/zowe/imperative/pull/852#pullrequestreview-1130452724_

**Comment by RASakach** [comment]
@zFernand0 Onboarding agrees this makes sense to introduce as a BEST PRACTICE for V3 conformance.  We will begin the process of creating the templates for V3 conformance and include this item.

**Comment by RASakach** [comment]
@zFernand0 :  I don't believe we added this to the conformance criteria for V3 (this was my fault!).  

Can we please revisit this and add it - from what I understand, this looks like a great addition.  (Since it is a "best practice" it can be added on a non-Version boundary).

**Comment by zFernand0** [comment]
> `@zFernand0` : I don't believe we added this to the conformance criteria for V3 (this was my fault!).
> 
> Can we please revisit this and add it - from what I understand, this looks like a great addition. (Since it is a "best practice" it can be added on a non-Version boundary).

No worries Rose,
We haven't heard of many plug-in developers making use of this feature 😋
